src/mender-auth/cli/actions.cpp

```diff
@@ -154,6 +154,10 @@ error::Error DaemonAction::Execute(context::MenderContext &main_context) {
 		return error::MakeError(error::ExitWithFailureError, "");
 	}
 
+	loop.Post([]() {
+		log::Info("Ready for authentication requests");
```

Comment:
S: `The authentication daemon is now ready to accept incoming authentication requests`